operations/package_repository_management/Manage_Repositories_with_Nexus.md

```diff
@@ -379,5 +397,39 @@ For example:
 # curl -sfkSL -X DELETE "https://packages.local/service/rest/v1/blobstores/NAME"
 ```
 
+### Authenticating to Access the REST API
+
+To access some of the REST API functions not listed above you need to authenticate using a username and password. This username and password is the same used to sign into the Web UI. Either the username and password of a properly permissioned Keycloak account or the nexus local admin account must be used. To get the nexus local admin account after a fresh install you can run the follow fuction:
```

Comment:
Minor wording change:

---

An authenticated username and password are required to access some of the REST API functions not listed above. This username and password are the same used to sign into the Web UI. Either the username and password of a properly permissioned Keycloak account or the Nexus local admin account must be used. 

Use the following function to get the nexus local admin account after a fresh install: